content/blog/news/2023/postdam/index.fr.md

```diff
@@ -0,0 +1,13 @@
+---
+title: "CEO Summit"
+linkTitle: "CEO Summit"
+date: 2023-06-01
+---
+
+Le 1er juin 2023 se tenait à Potsdam, près de Berlin, le CEO Summit du Memorandum of Understanding (MoU) entre SNCF et Deutsche Bahn.
+
+L'open source est l'un des huit sujets identifiés du MoU. Loïc Hamelin, Directeur du Programme, était invité à présenter OSRD devant JP Farandou et R Lutz, PDG des groupes SNCF et DB.
```

Comment:
See @axrolld previous comments. We shouldn't use "JP Farandou and R Lutz", maybe "M. Farandou" and "Dr. Lutz"?